timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/LocalPaxosComponents.java

```diff
@@ -50,12 +52,14 @@
 
     LocalPaxosComponents(TimelockPaxosMetrics metrics,
             PaxosUseCase paxosUseCase,
-            Path baseLogDirectory,
+            Path legacyLogDirectory,
+            Path sqliteLogDirectory,
             UUID leaderUuid,
             boolean canCreateNewClients) {
         this.metrics = metrics;
         this.paxosUseCase = paxosUseCase;
-        this.baseLogDirectory = baseLogDirectory;
+        this.baseLogDirectory = legacyLogDirectory;
+        this.sqliteLogDirectory = paxosUseCase.logDirectoryRelativeToDataDirectory(sqliteLogDirectory).toAbsolutePath();
```

Comment:
This should just be `sqliteLogDirectory`: we don't need to have separate SQLite instances for each of the use cases.